test/hack/checkLabels/main.go

```diff
@@ -103,32 +143,57 @@ func getMissingLabels() ([]string, error) {
 
 	// Get the list of labels with prefix "cancel_if_"
 	cancelLabels := getLabels(rulesList, "cancel_if_")
+	targetLabels, sourceLabels := getTargets(alertConf)
 
 	for _, cancelLabel := range cancelLabels {
-		var originLabel = strings.Split(cancelLabel, "cancel_if_")
-		if len(originLabel) < 2 {
-			return nil, fmt.Errorf("No origin label found in %q", cancelLabel)
+		var i = 0
+
+		for _, targetLabel := range targetLabels {
+			if cancelLabel == targetLabel {
+				i++
```

Comment:
Once i > 0, you can break out from the for loop

Reply:
True, I added it